recipes/pytest-regtest/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pytest-regtest" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 97fd19d12c44a8697be51faca4904112ae6e8460e798ac37a7c5639179bfc2c1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
`noarch: python` packages need their entry points listed.

Reply:
We don't understand what you mean exactly. There is no console_scripts entry point for pytest-regtest and the entrypoint for pytest11 itself works correctly, as show by the test.

Reply:
Hmmmm. I am not sure about entry points intended to be used by another package, which is the case here. I am pinging @conda-forge/core for guidance on this.

Reply:
kalefranz explains it here https://github.com/conda/conda-build/issues/2627#issuecomment-379815675 ... `entry_points` in meta.yaml reflect the `entry_points['console_scripts']` in the setup.py. Other entry_points don't need that handling, because they are not arch dependent... The recipe is very similar to pytest-cov which works the same way also regarding the entry_points https://github.com/conda-forge/pytest-cov-feedstock/blob/master/recipe/meta.yaml

Reply:
Ahhhhh ok! Sorry for the trouble and I learned something new! Thanks!